chsdi/templates/htmlpopup/unesco_bak.mako

```diff
@@ -4,9 +4,25 @@
 
 <%def name="table_body(c, lang)">
 <%
-    surface_ha = int(float(c['attributes']['bgdi_surface']) / 10000) 
+    surface_ha = round(c['attributes']['bgdi_surface'],2)
+    if lang == 'fr': 
+        name = 'name_fr'
+        zone = 'type_fr'
+    elif lang == 'de':
+        name = 'bgdi_name'
+        zone = 'type_de'
+    elif lang == 'it':
+        name = 'name_it'
+        zone = 'type_it'
+    elif lang == 'en':
+        name = 'name_en'
+        zone = 'type_en'
+    else:
+        name = 'name_rm'
+        zone = 'type_rm'
```

Comment:
minor but you could do:
`name = 'name_%s' % lang` and `zone = 'zone_%s' % lang`